### ISSUE TITLE
Is there support for exp function?

### ISSUE THREAD

**Issue by akirakotsugai** [waywaaard/skulpt_numpy#32]
It is not listed in the README.
Is there support for the exponential function?

Thanks a lot

**Comment by ebertmi** [comment]
Yes, see https://github.com/ebertmi/skulpt_numpy/blob/master/numpy/__init__.js#L2595
